docs/examples/user-configuration/user-config-template.yaml

```diff
@@ -1,145 +1,212 @@
-apiVersion: template.openshift.io/v1
+apiVersion: v1
 kind: Template
 labels:
-  template: user-config
-message: A new data grid service has been created in your project.
+  template: datagrid-service
```

Comment:
I think we should keep the name as `user-config` so that we don't confuse things with our datagrid-service.